pkg/controller/atlasproject/teams_test.go

```diff
@@ -438,3 +449,159 @@ func TestUpdateTeamState(t *testing.T) {
 		assert.Equal(t, 1, len(team.Status.Projects))
 	})
 }
+
+func Test_TeamGarbageCollect(t *testing.T) {
+	t.Run("Ensure team drops project ID when unassigned from a project", func(t *testing.T) {
+		logger := zaptest.NewLogger(t).Sugar()
+		workflowCtx := &workflow.Context{
+			Context: context.Background(),
+			Log:     logger,
+		}
+		testScheme := runtime.NewScheme()
+		testScheme.AddKnownTypes(akov2.GroupVersion, &akov2.AtlasProject{})
+		testScheme.AddKnownTypes(akov2.GroupVersion, &akov2.AtlasTeam{})
+		testScheme.AddKnownTypes(akov2.GroupVersion, &akov2.AtlasTeamList{})
+		testScheme.AddKnownTypes(akov2.GroupVersion, &corev1.Secret{})
```

Comment:
This whole block can be replaced with:
```suggestion
		akov2.AddToScheme(testScheme)
		corev1.AddToScheme(testScheme)
```